### PR TITLE
add information colorScheme for `Alert` & `Toast` & simplify `Alert`

### PR DESCRIPTION
**Pull request by jaf107**
resolves : https://github.com/optimizely-axiom/optiaxiom/issues/344

### CODE REVIEW

File: packages/react/src/toast/Toast.css.ts [hunk 1]
```diff
@@ -119,6 +119,11 @@ export const root = recipe({
           [accentColorVar]: theme.colors["red.300"],
         },
       }),
+      informational: style({
```
Comment (radnan) [inline packages/react/src/toast/Toast.css.ts:122]: this should be `information`
```suggestion
      information: style({
```

File: packages/react/src/toast/Toast.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ type ToastProps = BoxProps<
 
 const iconMap = {
   danger: IconCircleExclamationFilled,
+  informational: IconCircleInfoFilled,
```
Comment (toha993) [inline packages/react/src/toast/Toast.tsx:21]: this should be information

```suggestion
  information: IconCircleInfoFilled,
```

File: apps/docs/demos/alert/banner-usage/App.tsx [hunk 3]
```diff
@@ -7,12 +7,7 @@ export function App() {
   return (
     <>
       {open && (
-        <Alert
-          colorScheme="danger"
-          onClose={() => setOpen(false)}
-          variant="solid"
-          w="full"
-        >
+        <Alert colorScheme="danger" onClose={() => setOpen(false)} w="full">
```
Comment (toha993) [inline apps/docs/demos/alert/banner-usage/App.tsx:10]: I think we wont need banner usage anymore as we are exposing `banner` component individually.

File: apps/storybook/src/feedback/Toast.stories.tsx [hunk 4]
```diff
@@ -81,10 +82,10 @@ export const Appearance: Story = {
         </Button>
         <Button
           onClick={() =>
-            toaster.create(<Toast {...args} colorScheme="success" />)
+            toaster.create(<Toast {...args} colorScheme="informational" />)
```
Comment (toha993) [inline apps/storybook/src/feedback/Toast.stories.tsx:85]: this should be `information`
```suggestion
            toaster.create(<Toast {...args} colorScheme="information" />)
```

File: packages/react/src/alert/Alert.css.ts [hunk 5]
```diff
@@ -24,41 +22,19 @@ export const alert = recipe({
...
-        {
-          rounded: "md",
-        },
-        style({
-          backgroundColor: lightColorVar,
-        }),
-      ],
-      solid: style({
-        backgroundColor: solidColorVar,
+        backgroundColor: theme.colors["yellow.100"],
```
Comment (toha993) [inline packages/react/src/alert/Alert.css.ts:37]: Let's use sprinkles here

File: packages/react/src/alert/Alert.css.ts [hunk 6]
```diff
@@ -71,14 +47,23 @@ export const icon = recipe({
       mt: "4",
       w: "auto",
     },
+    style({
+      color: iconColorVar,
+    }),
```
Comment (toha993) [inline packages/react/src/alert/Alert.css.ts:52]: We can just use sprinkles in the variant, I think there's no need of keeping it in `var`

File: packages/react/src/alert/Alert.css.ts [hunk 7]
```diff
@@ -6,8 +6,6 @@ import {
   style,
 } from "../vanilla-extract";
 
-const solidColorVar = createVar();
-const lightColorVar = createVar();
 const iconColorVar = createVar();
```
Comment (toha993) [inline packages/react/src/alert/Alert.css.ts:9]:
```suggestion
```

File: packages/react/src/alert/Alert.css.ts [hunk 8]
```diff
@@ -71,14 +47,23 @@ export const icon = recipe({
...
       danger: style({
         vars: {
           [iconColorVar]: theme.colors["bg.error.solid.hover"],
         },
       }),
+      information: style({
+        vars: {
+          [iconColorVar]: theme.colors["brand.600"],
+        },
+      }),
```
Comment (toha993) [inline packages/react/src/alert/Alert.css.ts:66]: We can use sprinkles for each colorScheme, instead of keeping the value in `var`

**Comment by toha993** [comment]
https://github.com/optimizely-axiom/optiaxiom/blob/525151ef988354ea4fe88598166d5440888a45a4/apps/docs/demos/alert/appearance-usage/App.tsx#L8

Need to remove variant from docs.

File: apps/docs/demos/alert/banner-usage/App.tsx (hunk 3 above)
Comment (jaf107) [reply apps/docs/demos/alert/banner-usage/App.tsx:10]: let's just remove this altogether in that case

**Comment by jaf107** [comment]
> information: style({

File: apps/docs/demos/alert/banner-usage/App.tsx (hunk 3 above)
Comment (toha993) [reply apps/docs/demos/alert/banner-usage/App.tsx]: Yes, I meant that

File: apps/storybook/src/feedback/Toast.stories.tsx [hunk 9]
```diff
@@ -81,10 +82,10 @@ export const Appearance: Story = {
         </Button>
         <Button
           onClick={() =>
-            toaster.create(<Toast {...args} colorScheme="success" />)
+            toaster.create(<Toast {...args} colorScheme="information" />)
           }
         >
-          Success
+          Info
```
Comment (radnan) [inline apps/storybook/src/feedback/Toast.stories.tsx:88]:
```suggestion
          Information
```

**Review by radnan — APPROVED** [review]

**Pull request merged**